server/src/main/resources/templates/web/passcode.html

```diff
@@ -8,6 +8,19 @@
 <div layout:fragment="page-nav">
     <th:block layout:replace="nav :: nav"></th:block>
 </div>
+
+<div class="card warning" style=" width: 80%; margin: 0 auto; background-color: #febfc1; color: #ab2a2a; max-width: 800px; padding-top: 5vh; padding-bottom: 5vh; margin-bottom: 20px;">
+  <p class="h3-primary" style=" font-weight: 800; font-size: 22px; color: #ab2a2a;">
+    IMPORTANT: This hosting environment will be shut down this month.
+  </p>
+  <p class="h4-primary" style=" font-size: 18px; line-height: 25px; ">
+    If you haven't already migrated your app(s) to the new GovCloud hosting environment, please do so by March 15th.
```

Comment:
Same suggestion as above: let's link 'migrated your app(s)' to https://cloud.gov/docs/apps/govcloud/#migration, if there are no objections.